### DISCUSSION TITLE
Is there a way to make a Rule both atomic and slient?

### DISCUSSION THREAD

**Discussion by ccoffline** [pest-parser/pest#1008] [Q&A]
I try to parse a string literal just as `json5-rs` do. It has lots of `rule:char_literal` Pair in it.
https://github.com/callum-oakley/json5-rs/blob/5066f2e0a2726ceaeeb0a7786804787b3dc801f1/src/json5.pest#L23
How can i remove it? Is there any way to make it both atomic and slient?

**Comment by LeChatP** [comment]
#520
